doc/source/contributor-explanation-architecture.rst

```diff
@@ -1,6 +1,8 @@
 Flower Architecture
 ===================
 
+This document provides an overview of the Flower architecture. The architecture is designed to be modular and flexible, and can use two different types of engines: Edge Client Engine (ECE) and Virtual Client Engine (VCE).
```

Comment:
I'd suggest to already use the new naming in the comment, even if it's inconsistent with the diagram. I'm planning on updating the diagrams soon:
```suggestion
This document provides an overview of the Flower architecture. The architecture is designed to be modular and flexible, and can use two different types of engines: Deployment Engine and Simulation Engine.
```